### ISSUE TITLE
Slopes no longer visually appear on certain blocks

### ISSUE THREAD

**Issue by DanYami** [tModLoader/tModLoader#1020]
After the first version of TModloader was released on steam, I noticed that a few of my tiles appeared... odd, mainly my modded grass.

Collision-wise the slopes are still there, but they do not draw.

![image](https://user-images.githubusercontent.com/32519717/87716891-cebe3e80-c7af-11ea-85b2-62ebb2d3c705.png)
![image](https://user-images.githubusercontent.com/32519717/87716955-ed243a00-c7af-11ea-8b33-bcefdf7b5c7c.png)

if it helps in any way, these two tiles make use of glowmasks, which are correctly setup. They worked before the update with no issues

**Comment by Mirsario** [comment]
Can you make a minimal reproduction mod and attach the its source?

**Comment by Jofairden** [comment]
Yeah reproduction would be nice. Probably a small patch bug.

**Comment by DanYami** [comment]
After more investigation... I found out that what is happening here is that glowmasks simply are not affected by the sloping mechanic anymore, which they were previously. Here is a mod file with the grass (called Test Grass)

https://cdn.discordapp.com/attachments/213643426351153153/734163861896298597/TestMod.tmod

**Comment by DanYami** [comment]
https://cdn.discordapp.com/attachments/213643426351153153/737433510855311392/TestMod.zip here is the source, sent it to Jof at first but he prolly didn't see it

**Comment by JavidPack** [comment]
I've been testing this, and looking at tmod versions of the past, and I don't think this glowmask ever should have worked correctly as written. Are you positive slopped tiles with glowmasks worked on 0.11.6.2? I'm not seeing any code that would communicate to ModTiles about the various slope adjustments, those seem to all have to be done manually.

**Comment by DanYami** [comment]
Yes, I am certain this worked before, otherwise I wouldn't have noticed. I can supply some old screenshots to prove this once I am back home

**Comment by JavidPack** [comment]
Did you ever confirm this DanYami?
